srv/modules/runners/disk_led.py

```diff
@@ -0,0 +1,208 @@
+# -*- coding: utf-8 -*-
+"""
+This runner is used to turn on/off the ident and fault LED of a disk.
+"""
+from __future__ import absolute_import
+
+import logging
+import pprint
+import re
+
+import salt.client
+
+log = logging.getLogger(__name__)
+
+
+def _process(hostname, device_name, led, status):
+    """
+    Turn on/off the specified LED of the disk on the given host.
+    """
+    local_client = salt.client.LocalClient()
+    device_file = '/dev/{}'.format(device_name)
+    status = 'on' if status in [True, 'on'] else 'off'
+
+    # Get the configuration of the command to turn on/off the ident
+    # or fault LED on the target host.
+    cmd_cfg = local_client.cmd(
```

Comment:
This will fail due to the missing Makefile and deepsea.spec.in entries.

As already mentioned above, you are working with a distributed system where returns are not always guaranteed. It may be wise to validate output from external calls before passing them along.

Reply:
Done